src/Drupal/Commands/core/LocaleCommands.php

```diff
@@ -128,4 +133,168 @@ public function update($options = ['langcodes' => self::REQ])
 
         drush_backend_batch_process();
     }
+
+    /**
+     * Exports to a gettext translation file.
+     *
+     * @command locale:export
+     * @drupal-dependencies locale
+     * @option template To export the template file.
+     * @option langcode The language code of the exported translations.
+     * @option types String types to include, defaults to all types.
+     *      Types: 'not-customized', 'customized', 'not-translated'.
+     * @usage drush locale:export --langcode=nl > nl.po
+     *   Export the Dutch translations with all types.
+     * @usage drush locale:export --langcode=nl --types=customized,not-customized > nl.po
+     *   Export the Dutch customized and not customized translations.
+     * @usage drush locale:export --template > drupal.pot
+     *   Export the template file to translate.
+     * @aliases locale-export
+     * @see \Drupal\locale\Form\ExportForm::submitForm
```

Comment:
Nit-pick: Add empty line before `@see ...`. Makes it more readable as a separate subject. Not sure if any code style addresses this.